flink-core/src/test/java/org/apache/flink/core/memory/DataInputDeserializerTest.java

```diff
@@ -54,4 +54,14 @@ public void testAvailable() throws Exception {
         }
         Assert.assertEquals(0, dis.available());
     }
+
+    @Test
+    public void testReadWithLenZero() throws IOException {
+        byte[] bytes = new byte[] {};
```

Comment:
nit: might be simplified to `byte[] bytes = new byte[0]`

Reply:
fixed.